src/components/erc721/collectibles/collectibles_list.tsx

```diff
@@ -48,6 +53,7 @@ export const CollectiblesList = (props: Props) => {
         <MainContainer>
             <FiltersMenu>
                 <Title>{props.title}</Title>
+                <Button onClick={props.toggleCollectibleListModal}>Sell collectibles</Button>
```

Comment:
This button should be added on [this PR](https://github.com/0xProject/0x-launch-kit-frontend/pull/418), because in that PR the whole header is styled. Thoughts? @unjapones @gabitoesmiapodo

Reply:
It's merged now